tests/functional/conftest.py

```diff
@@ -11,15 +11,11 @@
 )
 
 
-@pytest.fixture(scope="session", autouse=True)
-def auto_use_dn(db_engine):
-    # TODO - Is this right?
-    return db_engine
```

Comment:
I'm not sure what this `auto_use_dn` (`db`?) fixture was for. I think it might have been trying to make the `db_engine` fixture, which doesn't itself have `autouse=True`, into an autoused fixture? But it isn't necessary because other fixtures, some of which are autouse, depend on `db_engine`, so `db_engine` is always used